### PR TITLE
Exec tachikoma update 20150312021435

### PR DESCRIPTION
**Pull request by tachikomapocket**
:hamster::hamster::hamster: <br /> <br /> Strategy bundler
Powered by <a href="http://tachikoma.io/?utm_source=github&utm_medium=pull_request&utm_campaign=build5981608014988773760">Bundle Update as a Service - Tachikoma.io</a>

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,5 +1,5 @@
 GEM
-  remote: http://rubygems.org/
+  remote: https://rubygems.org/
```
Comment (masutaka) [inline Gemfile.lock]: tachikoma先生も直してくれてたのか

**Comment by masutaka** [comment]
* execjs: https://github.com/sstephenson/execjs/compare/v2.3.0...v2.4.0
* listen: https://github.com/guard/listen/compare/v2.8.5...v2.9.0
* thread_safe: https://github.com/ruby-concurrency/thread_safe/compare/v0.3.4...v0.3.5

※ [compare_linker](https://rubygems.org/gems/compare_linker)を使って手動で書き込んでいます

**Comment by tmd45** [comment]
:+1:

**Comment by masutaka** [comment]
thx!

**Pull request merged**